rpc/src/module/net.rs

```diff
@@ -586,7 +587,7 @@ impl NetRpc for NetRpcImpl {
             .connected_peers()
             .iter()
             .map(|(peer_index, peer)| {
-                let mut addresses = vec![&peer.connected_addr];
+                let mut addresses: HashSet<_> = vec![&peer.connected_addr].into_iter().collect();
```

Comment:
no need to create vec and convert to hashet, we may use the iterator to collect directly:
```
let addresses: HashSet<_> = std::iter::once(peer.connected_addr).chain(peer.listened_addrs.iter()).collect();
```